packages/okta-react/README.md

```diff
@@ -267,6 +266,55 @@ class App extends Component {
 export default App;
 ```
 
+#### PKCE Example
+
+Assuming you have configured your application to allow the `Authorization code` grant type, simply pass `pkce=true` to the `Security` component. This will configure the `Auth` object to perform PKCE flow for both login and token refresh.
+
+```typescript
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Security issuer='https://{yourOktaDomain}.com/oauth2/default'
+                  clientId='{clientId}'
+                  pkce=true
```

Comment:
Should this not be `{true}`?  IIRC JSX props are either a braced expression or a string literal, no boolean.